tests/integration_test.sh

```
@@ -32,5 +32,37 @@ $CLIENT_BIN "TIME" | grep -q "20" || fail "TIME did not return expected format"
+GET interactive
+DEL interactive
+TIME
+PING
+exit
+EOF
+
+# test bad command definition
+$CLIENT_BIN "SET test" 2>&1 | grep -q "ERROR" || fail "Bad command did not return ERROR"
+
+# test invalid command
+$CLIENT_BIN $(for i in {1..500}; do echo -n "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA "; done) 2>&1 | grep -q "Invalid command" || fail "Invalid command did not return ERROR"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)